invenio_theme/templates/invenio_theme/500.html

```diff
@@ -27,6 +27,6 @@
 {% block message %}
   <h1><i class="fa fa-flash"></i> {{_('Internal server error')}}</h1>
   {% if g.sentry_event_id %}
-  <p><small class="text-muted">{{_('Error identifier')}}: {{ g.sentry_event_id }}</small></p>
+  <p><small class="text-muted">{{_('Identifier error')}}: {{ g.sentry_event_id }}</small></p>
```

Comment:
I think the meaning here is something akin to "Error's Identifier" or ``Error ID`` (followed by the ID number of the error).
I'd go with the original or perhaps something more like ``Error ID``.